graphql/schema/types/config.graphql

```diff
@@ -155,6 +156,7 @@ type AdminSettings {
   podLifecycle: PodLifecycleConfig
   scheduler: SchedulerConfig
   repotracker: RepotrackerConfig
+  web: Web
```

Comment:
I don't know that we want to introduce a new field rather than just using the existing ones in admin settings
```suggestion
  api: APIConfig
  ui: UIConfig
```